### ISSUE TITLE
OpenFace as AU regressor in network to be trained

### ISSUE THREAD

**Issue by aiueogawa** [albertpumarola/GANimation#72]
@albertpumarola 
Hi, thanks for your great work!

In paper, AU regressor is used to evaluate the AU conditioning of a generator.
This shares parameters with an image critic like ACGAN.
Why not use OpenFace as AU regressor?

I think using OpenFace without any trainable parameters leads to more accurate conditioning of AU.
How do you think of this idea?

UPDATE: 
If OpenPose is composed of only fully differenciable components, it is possible to backpropagate gradients.

**Comment by albertpumarola** [comment]
I wanted the model to fit in one GPU, so the AU regressor had to be very small. But yes, as long as the AU regressor is differentiable it should be fine.

**Comment by albertpumarola** [comment]
Closed for inactivity
